my-app/src/actions/donateAction.js

```diff
@@ -0,0 +1,14 @@
+export const createDonate = (project) =>{
```

Comment:
It would be better to rename "project" to "payment" or other name according to the actions performed. Only for better code rewiew.